jpa-container/src/main/java/org/apache/aries/jpa/container/impl/AriesEntityManagerFactoryBuilder.java

```diff
@@ -83,6 +83,7 @@
 
 	private boolean complete;
 
+	private Map<String, Object> overrides;
```

Comment:
if i get it right this is normally set only once at a time (when the config changes), so maybe make it an AtomicReference ?